openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -15,6 +18,60 @@ function update_len() {
     $('#excerpts-excerpt-len').html(2000 - len).css('color', color);
 }
 
+/**
+ * This is needed because jQuery has no forEach equivalent that works with jQuery elements instead of DOM elements
+ * @param selector - css selector used by jQuery
+ * @returns {*[]} - array of jQuery elements
+ */
+function getJqueryElements(selector){
+    const queryResult = $(selector);
+    const jQueryElementArray = [];
+    for (let i = 0; i < queryResult.length; i++){
+        jQueryElementArray.push(queryResult.eq(i))
+    }
+    return jQueryElementArray;
+}
+
+export function initLanguageMultiInputAutocomplete() {
+    $(function() {
+        getJqueryElements('.multi-input-autocomplete--language').forEach(jqueryElement => {
```

Comment:
as stated before, `$().each` is probably a better solution here, particularly given you run `$(selector)` on every element. Creating a collection once will be much more efficient.

I've pushed forward with the patch as I think getting this out of the HTML is the priority here but feel free to explore this (or not) in a follow up.